routes/favicon.coffee

```diff
@@ -0,0 +1,28 @@
+request = require 'request'
+fs = require 'fs'
+cachefile = 'favicon.ico'
+
+favicon = (req, res, next) ->
+	try
+		res.send fs.readFileSync cachefile
+	catch error
+		request "https://vine.co/", (error, response, body) ->
```

Comment:
Come to think of it, these read and write file calls should be using the async versions rather than the sync ones. Are you able to make that change? Let me know if you need some guidance.

While you're at it, can you make the `cachefile` variable in caps (to show it's meant as a constant) and separate it from the required modules by a line.

And where is that saving to? The current working directory, I'd assume, which could be anywhere the program is run from. I'm not sure this is a good idea. It'd probably be best to lock this to a specific place either alongside the vinefeed program itself (use `__dirname` and go from there) if you feel that makes sense, or a temporary directory elsewhere on the system (maybe try [os.tmpdir](https://nodejs.org/api/os.html#os_os_tmpdir)).